pgd_splicer/ProcessPDBTask.py

```diff
@@ -405,8 +405,48 @@ def parseWithBioPython(path, props, chains_filter=None):
     # Be kind; rewind.
     decompressed.seek(0)
 
+    alt_structure = Bio.PDB.PDBParser().get_structure('pdbname',
+                                                  decompressed.name)
+    io=PDBIO()
+    io.set_structure(alt_structure)
+    io.save("not-ordered.pdb")
```

Comment:
Use a temporary file here.  There's a good example right above this section of code.